packages/fxa-settings/src/models/integrations/relier-interfaces.ts

```diff
@@ -44,8 +44,23 @@ export interface RelierCmsInfo {
   clientId: string;
   entrypoint: string;
   shared?: SharedRelierCmsInfo;
+
   EmailFirstPage?: PageRelierCmsInfo
+
   SignupSetPasswordPage?: PageRelierCmsInfo
   SignupConfirmCodePage?: PageRelierCmsInfo
   SignupConfirmedSyncPage?: PageRelierCmsInfo
+
+  SigninPage?: PageRelierCmsInfo
+  SigninTotpCodePage?: PageRelierCmsInfo
+  SigninTokenCodePage?: PageRelierCmsInfo
+  SigninUnblockCodePage?: PageRelierCmsInfo
+
+  // SigninRecoveryPhone?: PageRelierCmsInfo
+  // SigninRecoveryCode?: PageRelierCmsInfo
+  // SigninRecoveryChoice?: PageRelierCmsInfo
+  // SigninPushCodeConfirm?: PageRelierCmsInfo
+  // SigninPushCode?: PageRelierCmsInfo
+  // SigninConfirmed?: PageRelierCmsInfo
+  // CompleteSignin?: PageRelierCmsInfo
```

Comment:
Looking at list of pages mentioned in the PR description, should these be uncommented?

Reply:
I remove these. We don't have finalized UX, so only the default button color will be updated.